What it happens when we run the code us that there is an error in line 8 telling us that there is an operation which is divided by zero, therefore there is an exception of class 'ZeroDivisionError' so as to solve it we have to put inside the function 'except ZeroDivisionError', this happens when a and b are the same.